addons/web/static/src/views/form/form_controller.scss

```diff
@@ -712,6 +712,13 @@
         }
     }
 
+    .o_field_number, .o_field_monetary {
```

Comment:
this seems maybe too broad, I think this will affect any monetary field, so for example also in readonly/non-readonly field anywhere on the form view (the commit only speak of total and subtotal)

if it's intended, the previous fixes CSS could probably be partially removed since the change should already be applied by this fix (it seems when there is .o_list_number class, there is already .o_field_monetary or .o_field_number)

Reply:
it is infact intended to have montary fields always display the output in direction ltr, independantly of the direction if the language , but text-align I think might change , so maybe I will split the change of the css.